frontend/components/RegionsContainer.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import fabric from '../services/fabric';
+import * as propShapes from '../prop-shapes';
+import Regions from './Regions';
+import { selectedFace } from '../redux/selectors';
+import { requestFaces } from '../redux/actions';
+
+class RegionsContainer extends PureComponent {
+  componentDidMount() {
+    this.props.onEnter();
+  }
+  componentDidUpdate() {
+  }
```

Comment:
We can remove this empty block; looks like there's some other lint issues too. Code looks good so 👍  once those are resolved!